src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -24,3 +24,30 @@ <h2 class="card-pf-title">
     </p>
   </div>
 </div>
+<div ng-if="isMiniLayout" class="card-pf card-pf-aggregate-status card-pf-aggregate-status-mini" ng-class="{'card-pf-accented': shouldShowTopBorder}">
+  <h2 class="card-pf-title">
+    <span ng-if="status.iconClass" class="{{status.iconClass}}"></span>
+    <a ng-if="status.href" href="#{{status.href}}">
+      <span class="card-pf-aggregate-status-count">{{status.count}}</span>
+      {{status.title}}
+    </a>
+    <span ng-if="!status.href">
+      <span class="card-pf-aggregate-status-count">{{status.count}}</span>
+      {{status.title}}
+    </span>
```

Comment:
Need to document the status parameter and add that when using the 'mini' layout that status is used rather than notifications.